src/cp.ts

```diff
@@ -1,10 +1,12 @@
-import fs from 'node:fs';
+import fs, { rmSync } from 'node:fs';
 import { WritableStreamBuffer } from 'stream-buffers';
-import * as tar from 'tar';
```

Comment:
There are some linter errors in this file.

Reply:
fixed.